src/main/java/hudson/plugins/gradle/GradleInstallationProvider.java

```diff
@@ -0,0 +1,15 @@
+package hudson.plugins.gradle;
+
+/**
+ * Interface for providing {@link GradleInstallation} independent of build type.
+ * 
+ * @author Sönke Küper
+ */
+public interface GradleInstallationProvider {
+
+    /**
+     * Provides all available {@link GradleInstallation}.
+     */
+    public abstract GradleInstallation[] getInstallations();
```

Comment:
```suggestion
    GradleInstallation[] getInstallations();
```

`public` and `abstract` are implied by being in an interface.